pkgs/top-level/all-packages.nix

```diff
@@ -38152,4 +38152,6 @@ with pkgs;
   aitrack = libsForQt5.callPackage ../applications/misc/aitrack { };
 
   widevine-cdm = callPackage ../applications/networking/browsers/misc/widevine-cdm.nix { };
+
+  ucblogo = callPackage ../development/interpreters/ucblogo { };
```

Comment:
Putting it on the bottom is nearly guaranteed to result in a merge conflict. I reccomend moving this attribute to a line somewhere in the middle of the file.